include/ignition/gui/qml/PluginMenu.qml

```diff
@@ -104,6 +116,7 @@ Popup {
     model: MainWindow.PluginListModel()
 
     delegate: ItemDelegate {
+      property variant myData: model
```

Comment:
Instead of `myData` let's have a more meaningful name, eg. `pluginModel`

Reply:
Fixed in 8cc78c8